src/styles/vars.scss

```
@@ -0,0 +1,12 @@
+$header-text-color: #060b35;
+$header-text-color-hover: #00acdc;
+$card-bg: #fff;
+$card-border-color: #f3f3f3;
+$button-buy-color: #00acdc;
+$card-text-color: #616070;
+$card-h-color: #060b35;
```

Don't need to duplicate variables with same values
```suggestion
$card-text-color: #616070;
```